doc/network_file_desc.rst

```diff
@@ -0,0 +1,200 @@
+:orphan:
+
+.. _network_file_content:
+
+====================
+Network File Content
+====================
+
+File Content Description
+------------------------
+hdf5 is the file format used for storing the Network object. The network is stored in a layered format. The first layer consists of the network attributes.
```

Comment:
```suggestion
hdf5 is the file format used for storing the Network object. The network is stored in a multi-level format. The first level consists of the network attributes.
```

Using "layer" as the terminology might get confusing, as there is the biologically "layers" of the model